pkg/identityserver/end_device_registry.go

```diff
@@ -98,8 +99,10 @@ func (is *IdentityServer) createEndDevice(ctx context.Context, req *ttnpb.Create
 }
 
 func (is *IdentityServer) getEndDevice(ctx context.Context, req *ttnpb.GetEndDeviceRequest) (dev *ttnpb.EndDevice, err error) {
-	if err = rights.RequireApplication(ctx, req.EndDeviceIdentifiers.ApplicationIdentifiers, ttnpb.RIGHT_APPLICATION_DEVICES_READ); err != nil {
-		return nil, err
+	if err := clusterauth.Authorized(ctx); err != nil {
```

Comment:
This should not be necessary. The IS already has middleware that assigns `ttnpb.AllClusterRights` when using cluster auth.